### PR TITLE
Add note about mapping a single DBRP to a single bucket

### PR DESCRIPTION
**Pull request by sanderson**
Closes influxdata/DAR#298

- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.2/reference/api/influxdb-1x/dbrp.md [hunk 1]
```diff
@@ -63,12 +67,19 @@ the DBRP mapping service checks for a bucket mapped to the database and retentio
 
 ### When querying data
```
Comment (jstirnaman) [inline content/influxdb/v2.2/reference/api/influxdb-1x/dbrp.md:68]:
```suggestion
### Query mapped buckets
```

**Review by jstirnaman — APPROVED** [review]
LGTM - a minor suggestion and a point of confusion.

File: content/influxdb/v2.2/query-data/influxql.md [hunk 2]
```diff
@@ -212,7 +219,7 @@ To return results as **CSV**, include the `Accept: application/csv` header.
 
 ## InfluxQL support
 
-InfluxDB OSS {{< current-version >}} supports InfluxQL **read-only** queries. See supported and unsupported queries below.
+InfluxDB {{< current-version >}} supports InfluxQL **read-only** queries. See supported and unsupported queries below.
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/influxdb/v2.2/query-data/influxql.md]: I'll just remove read-only

File: content/influxdb/v2.2/reference/api/influxdb-1x/dbrp.md (hunk 1 above)
Comment (sanderson) [reply content/influxdb/v2.2/reference/api/influxdb-1x/dbrp.md:68]: With this being a reference doc that describes the behaviors of DBRP maps, imperative voice doesn't feel right here.

**Pull request merged**